README.md

```diff
@@ -92,22 +92,42 @@ To use the Material theme, import the correspondent file from the `theme/materia
   `vaadin-element.html`
 
 
-## Running demos and tests in browser
+## Running demos and tests in a browser
+Ensure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
 
-1. Fork the `vaadin-element` repository and clone it locally.
+Clone the `vaadin-select` repository.
+```bash
+$ git clone https://github.com/vaadin/vaadin-select.git
+```
+
+Change into the `vaadin-select` directory.
+```bash
+$ cd vaadin-select
+```
+
+Install the dependencies.
+```bash
+$ npm install
+$ bower install
+```
+
+Launch a webserver to serve the demo & tests.
+```bash
+$ polymer serve --port 3000
+```
+
+View the demos at :
```

Comment:
nit: why do you add whitespace before colons? That's not what we usually do. Please remove here and below